server/strategy.go

```diff
@@ -874,8 +851,21 @@ func repairOps(c *cke.Cluster, cs *cke.ClusterStatus, constraints *cke.Constrain
 	return ops, phaseRepair
 }
 
-func rebootOps(c *cke.Cluster, constraints *cke.Constraints, rebootArgs DecideOpsRebootArgs, nf *NodeFilter) (ops []cke.Operator) {
-	if len(rebootArgs.RQEntries) == 0 {
+func rebootOps(c *cke.Cluster, cs *cke.ClusterStatus, constraints *cke.Constraints, nf *NodeFilter) (ops []cke.Operator) {
+	if len(cs.RebootQueue.RebootCancelled) > 0 {
```

Comment:
With this refactor, some values are saved to RebootCancelled and RebootDequeued even when the reboot queue is disabled.